components/client/src/main/kotlin/com/hotels/styx/client/connectionpool/LatencyTiming.kt

```diff
@@ -46,7 +46,27 @@ object LatencyTiming  {
      * @param context context that can have timers set in it
      */
     @JvmStatic
-    fun startResponseTiming(metrics: CentralisedMetrics?, context: HttpInterceptor.Context?) {
+    fun startResponseTiming(
+        metrics: CentralisedMetrics?,
+        context: HttpInterceptor.Context?,
+    ) {
+        if (metrics != null && context is TimeMeasurable) {
+            context.timers?.startTiming(metrics, RESPONSE_PROCESSING)
+        }
+    }
+
+    /**
+     * Stops the response processing timer, if one is present in the context.
+     * Note that this only has an effect if the metrics object is non-null and the context is [TimeMeasurable].
+     *
+     * @param metrics metrics - required to access timers to start them
+     * @param context context that can have timers set in it
+     */
+    @JvmStatic
+    fun stopResponseTiming(
+        metrics: CentralisedMetrics?,
+        context: HttpInterceptor.Context?,
+    ) {
         if (metrics != null && context is TimeMeasurable) {
             context.timers?.startTiming(metrics, RESPONSE_PROCESSING)
```

Comment:
The `stopResponseTiming` method seems to be calling the `startTiming` one, is that intended?

Reply:
oh, shoot, let me fix it

Reply:
done